src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+Declaration(Class(<http://purl.obolibrary.org/obo/CL_4033009>))
+Declaration(Class(<http://purl.obolibrary.org/obo/CL_4033010>))
+Declaration(Class(<http://purl.obolibrary.org/obo/CL_4033011>))
+Declaration(Class(<http://purl.obolibrary.org/obo/CL_4033012>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0002339>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0004849>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0010397>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0011184>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0012398>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0012399>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_0012419>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_8410041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_8410042>))
+Declaration(Class(<http://purl.obolibrary.org/obo/UBERON_8410043>))
+Declaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>))
+Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/IAO_0000115>))
+Declaration(AnnotationProperty(<http://purl.org/dc/terms/contributor>))
+Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#hasDbXref>))
+
+
+
+############################
+#   Classes
+############################
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033000> (blood vessel endothelial cell of venule of lymph node)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.3389/fimmu.2017.00045"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28217126"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033000> "A(n) blood vessel endothelial cell that is part of a(n) venule of lymph node."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033000> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033000> "blood vessel endothelial cell of venule of lymph node"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033000> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000071> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410041>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033001> (blood vessel endothelial cell of arteriole of lymph node)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.3389/fimmu.2017.00045"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28217126"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033001> "A(n) blood vessel endothelial cell that is part of a(n) arteriole of lymph node."^^xsd:string)
```

Comment:
Did not find this specific cell type explicitly mentioned in the dbxref.

Reply:
@aleixpuigb, do you want to comment?

Reply:
There is no mention to blood vessel endothelial cells in venules/arteries of the lymph node apart of the high endothelial cells in high endothelial venules (HEVs) in the literature. However there is reference of the artery and venules (non-HEVs), and according to literature, [all blood vessels have blood vessel endothelial cells](https://www.ncbi.nlm.nih.gov/books/NBK26848/#:~:text=Endothelial%20Cells%20Line%20All%20Blood%20Vessels&text=The%20amounts%20of%20connective%20tissue,endothelial%20lining%20is%20always%20present.):

"The amounts of connective tissue and smooth muscle in the vessel wall vary according to the vessel's diameter and function, but the endothelial lining is always present. "

I agree that the dbxref provided is not explicit about the existence of those cells. What would be your recommendation  for cells that are not mentioned in literature, but the existence is implicit, @bvarner-ebi?

Reply:
I don't doubt that these cells exist.  I think reasonable inference is fine - blood vessels have endothelial cells & these blood vessels exist.  Would be strengthened by mention of markers for the endothelial layer - but I think OK without.  r just as Alex has used here. 

As these  terms come from ASCT+B tables, there are multiple expert authors.  We could try to get their ORCiDs.  Or we could just add a comment on origins.